### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by j-j-gajjar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,19 +7,18 @@ interesting method.
...
-field correspond to coordinates in the image.
+lot correspond to coordinates in the image.
 
 "Balls" are introduced into the field.  The radius of the balls is
-proportional the intensity of the image at the corresponding coordianate.
+proportional to the intensity of the image at the corresponding coordinate.
 The balls are given some initial velocity, and there is some friction to
-slow them down.  The balls also have a repulsive magnetic field which 
-varies in strength that is again proportional to the radius of the ball
+slow them down.  The balls also have a repulsive magnetic field which varies in strength that is again proportional to the radius of the ball
```
Comment (smcameron) [inline README.md]: The removal of the newline here seems incorrect.

File: README.md [hunk 2]
```diff
@@ -7,19 +7,18 @@ interesting method.
 The method is as follows:
 
 A rectangular field is mapped to the image so that coordinates in the
-field correspond to coordinates in the image.
+lot correspond to coordinates in the image.
```
Comment (smcameron) [inline README.md]: This change s/field/lot/ seems incorrect.

**Comment by j-j-gajjar** [comment]
please add "hacktoberfest-accepted" into the label

**Comment by smcameron** [comment]
It's not necessary, from https://hacktoberfest.digitalocean.com/resources/maintainers

> "A pull request can be approved *either* by giving it an overall approving review from maintainers, *or by merging the pull request*, or adding the 'hacktoberfest-accepted' label"

Since it's merged, this is good enough.  The "hactoberfest-accepted" label is the weakest form of acceptance.  Merging is stronger approval.

It should eventually show up for you at https://hacktoberfest.digitalocean.com/profile

**Pull request merged**